src/posts/2020-10-14-wcag2.md

```diff
@@ -0,0 +1,96 @@
+  - url:  https://en.wikipedia.org/wiki/Web_Content_Accessibility_Guidelines
+    title: Web Content Accessibility Guidelines
+    source: Wikipedia
+date: 2020-10-14
+eleventyNavigation:
+  key: {{ category }}
+  title: {{ title }}
+tags:
+  - background
+---
+The **[Web Content Accessibility Guidelines**](https://www.w3.org/WAI/standards-guidelines/wcag/) (WCAG) are part of a series of web accessibility guidelines published by the **[Web Accessibility Initiative](https://www.w3.org/WAI/about/)** (WAI) . The WCAGs goal is to provide a single shared standard for web content accessibility that meets the needs of individuals, organizations and governments internationally. 
+
+## WCAG 2.0
+
+WCAG 2.0 was published on 11 december 2008 and consists of 12 to 13 guidelines. It is organized around four principles which form the acronym `POUR`.  They include:
+
+1. Content must be `perceivable (P)`.
+
+- Provide a text alternative for all non-text content.
+- Provide synchronized alternatives for multimedia (such as captioned video, audio descriptions and so forth).
+- Information and structure must be separate from presentation.
+- Make it easy to distinguish foreground information from the background (for example good color contrast).
+
+2. Interfaces must be `operable (O)`. 
+
+- All functionality must be operable via keyboard.
+- All users must be able to control limits on their reading or interaction.
+- Users must be to avoid content that can cause seizures due to photosensitivity.
+- Provide mechanisms for users to find content, orientate themselves and navigate through it.
+- Help users avoid  mistakes and make it easier to correct mistakes when they occur.
+
+3. Content and controls must be `understandable (U)`.
+
+- Make text content readable and understandable.
+- Make the placement and functionality of content predictable.
+
+4. Content must be `robust (R)`.
+
+- Support compatibility with current and future user agents.
+- Ensure that content is accessible, or provide accessible alternatives.
+
+### Layers of guidance
+
+WCAG 2.0 provides various ways to help authors meet its standard called `layers of guidance`. They include the following :
+
+- Guidelines.
+- Success criteria.
+- Techniques.
+- Common failures.
+
+### Guidelines
+
+WCAG 2.0 guidelines are categorized into 3 [levels of conformance](https://www.w3.org/WAI/WCAG21/Understanding/conformance#levels) in order to meet the needs of different groups and situations. Under each principle, there is a [list of guidelines](https://www.w3.org/TR/WCAG20/#guidelines) that addresses the principles. 
+
+### Success criteria
+
+Under each guideline, there are success criteria that describe specifically what is to be done in order to conform to the standard.  Each success criteria is written as a statement that will be either true or false when specific web content is tested against it.  A convenient [list of the success criteria](https://www.w3.org/WAI/WCAG21/quickref/) can be found here.
```

Comment:
```suggestion
Under each guideline, there are success criteria that describe specifically what is to be done in order to conform to the standard.  Each success criteria is written as a statement that will be either true or false when specific web content is tested against it.  A convenient [list of the success criteria](https://www.w3.org/WAI/WCAG21/quickref/) is also provided.
```

Updating this to make where "here" is less unclear.

Reply:
Okay.